components/schemas.yml

```diff
@@ -65,13 +65,15 @@ Account:
 DecoratedSSHKeys:
   allOf:
     - $ref: '#/WrappedSSHKeys'
+    - $ref: 'schemas/pages.yml#/Pagination'
     - $ref: '#/Meta'
   example:
     ssh_keys:
       - id: 512189
         fingerprint: 3b:16:bf:e4:8b:00:8b:b8:59:8c:a9:d3:f0:19:45:fa
         public_key: "ssh-rsa AEXAMPLEaC1yc2EAAAADAQABAAAAQQDDHr/jh2Jy4yALcK4JyWbVkPRaWmhck3IgCoeOO3z1e2dBowLh64QAM+Qb72pxekALga2oi4GvT+TlWNhzPH4V example"
         name: My SSH Public Key
+    links: {}
```

Comment:
This doesn't seem to override the links examples here. The output of `swagger-cli bundle` still has the examples with `images` in the path. Looks like this works:

```
    - $ref: 'schemas/pages.yml#/Pagination'
      example: {}
```

Reply:
Nice catch!